x-pack/plugins/security/public/views/management/users_grid/_users.scss

```diff
@@ -0,0 +1,8 @@
+// HACK -- Fix for background color full-height of browser
```

Comment:
I just double-checked and these hacks are no longer necessary. Can you just remove this file?